pages/ksphere/kommander/latest/clusters/attach-cluster/index.md

```diff
@@ -8,13 +8,17 @@ excerpt: A guide for attaching an existing Kubernetes cluster using kubeconfig
 
 ## Attach Kubernetes Cluster
 
-Using the **Add Cluster** option, you can attach an already existing cluster, or an existing Konvoy cluster, directly to Kommander. You can access the multi-cluster management and monitoring benefits Kommander provides while keeping your existing cluster on its current provider and infrastructure.
+You can attach an existing cluster directly to Kommander. There are two main ways of doing it, described below, and one requires to create a new service account. If the cluster you wish to attach has been created using Amazon EKS, Azure AKS, or Google GKE, please follow the instructions "Attaching Kubernetes Cluster Using a New Service Accoun".
```

Comment:
> and one requires to create a new service account

This builds some kind of suspense where I don't think it's necessary. As a reader coming to this page I have a very specific problem I want to solve, which is attaching a cluster so the information how I'd have to do it is not important in this paragraph.